test/spec/modules/conversantAnalyticsAdapter_spec.js

```diff
@@ -0,0 +1,733 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
+import {getGlobal} from 'src/prebidGlobal.js';
```

Comment:
Not used.

Reply:
Used to stub out getGlobal to help with testing

Reply:
Yeah, but doesn't seem to be needed when I tested it.  The stub setup doesn't needed it because it already has a reference to prebidGlobal.

Reply:
Doesn't apply anymore anyway based on my updates